docs/src/modules/utils/generateMarkdown.js

```diff
@@ -344,7 +363,7 @@ function generateClasses(reactAPI) {
   }
 
   if (!reactAPI.styles.name) {
-    throw new Error(`Missing styles name on ${reactAPI.name} component`);
+    styleNameError(reactAPI.name);
```

Comment:
Why do we need this indirection? Be aware that this adds another line to the stack trace.

Reply:
Good point. The generateClasses() and generateName() functions both throw the same custom error, and I wasn't sure on the preference for avoiding repetition, so I wrapped it in a function.